Google.PowerShell/Provider/GoogleCloudStorageProvider.cs

```diff
@@ -480,14 +481,19 @@ protected override void GetChildItems(string path, bool recurse)
                             {
                                 GetChildItems(bucket.Name, true);
                             }
-                            catch (Exception e)
+                            catch (GoogleApiException e) when (e.HttpStatusCode == HttpStatusCode.Forbidden) { }
+                            catch (AggregateException e)
                             {
-                                if ((e as GoogleApiException)?.HttpStatusCode != HttpStatusCode.Forbidden)
+                                foreach (Exception innerException in e.InnerExceptions)
                                 {
-                                    WriteError(
-                                        new ErrorRecord(e, null, ErrorCategory.NotSpecified, bucket.Name));
+                                    WriteError(new ErrorRecord(
```

Comment:
How long is the next line? I cannot even see it right in GitHub's side-by-side view.

Perhaps declare `var error = new ErrorRecord(` as a separate local variable, and spread its parameters across a couple of lines?